### ISSUE TITLE
Support sub commands?

### ISSUE THREAD

**Issue by onmyway133** [tldr-pages/tldr#523]
Hi,

I 'd like to `tldr git commit`

I use the `npm` client

But it is not possible at the moment

**Comment by ostera** [comment]
Hi! http://www.ostera.io/tldr.jsx/#/git-commit

`git` is composed by a variety of smaller programs, like `git-add` or `git-commit`. Following that, you can easily search all git tldr pages by using a `-` instead of a space!

Granted, some clients could allow the use of a space and show the appropriate page, but that is an issue for the clients repositories.

Thanks for taking the time to file an issue, and I hope this was of help!

**Comment by onmyway133** [comment]
@ostera I use `npm` client. And thanks. It does help :+1: 

I just had an issue. When I open Terminal, and type `tldr git`, it says `Not found. Try tldr --update`

I open Terminal again and I don't see that issue any more. And I can't reproduce it anymore

**Comment by igorshubovych** [comment]
So run 

~~~
tldr --update
~~~

**Comment by onmyway133** [comment]
@igorshubovych Running `tldr --update` does not help. After I reopen Terminal, it works again

**Comment by igorshubovych** [comment]
@onmyway133 
can you please tell me more about your bug?

which OS do you have?
which TLDR client do use?
if on OSX/Linux run the following commands and share the output:

```shell
which tldr
ls -la ~/.tldr
cat ~/.tldrrc
```

**Comment by igorshubovych** [comment]
Upgrade to tldr-node-client 1.6.0 and run
```bash
$ tldr git commit
```
